src/Disputes/Grid/GridContent/index.js

```diff
@@ -34,14 +34,14 @@ class GridContent extends Component {
         <div className='items'>
           {
             disputes.map(dispute =>
-              <Link to={`disputes/${dispute.caseId}`}>
-                <div key={dispute.caseId} className='items-row'>
+              <Link key={dispute.hash} to={`disputes/${dispute.hash}`}>
```

Comment:
I think caseId is linked to the `disputeId` not the hash. See https://github.com/kleros/kleros/blob/master/contracts/KlerosPOC.sol#L385.